### ISSUE TITLE
Examples for Expand acronyms on their first use

### ISSUE THREAD

**Issue by iadawn** [w3c/wai-quick-start#135]
Tip: [Expand acronyms on their first use](http://w3c.github.io/wai-quick-start/writing.html#expand-acronyms-on-their-first-use)

**Comment by AndrewArch** [comment]
Example is fine, but most screen reader users I know don't want to hear it expanded (via the abbr elements) on subsequent uses - becomes noise for them like it would visually for us :( and they would rather just CVAA on subsequent uses.

**Comment by iadawn** [comment]
From @bakkenb:

> I think an example is needed. I like the current example and the tip text itself. I think they are very clearly stated.

**Comment by vmmiller** [comment]
I like the example provided.

**Comment by sharronrush** [comment]
Example is good

**Comment by slhenry** [comment]
I think we do _not_ need the clutter of a separated out example. [medium-strong] Most people will know that this means without any example. I think we can just have in the description: "for example, Web Content Accessibility Guidelines (WCAG)"

IMPORTANT: I think we cannot use the text of this example. It could be taken as WAI weighing in on policy-related issues.

**Comment by iadawn** [comment]
Agreed to include an example, but not making reference to CVAA
